src/clr-angular/wizard/wizard-stepnav-item.ts

```diff
@@ -66,6 +66,13 @@ export class ClrWizardStepnavItem {
     return this.pageCollection.previousPageIsCompleted(this.page);
   }
 
+  public get _disabledButton(): string | null {
```

Comment:
Do we need a new public getter here? It's just a ternary that's already referencing the component's API. Would it be better to make it `[attr.disabled]="this.isDisabled ? '' : null"`? Or is there a reason we need it to be a getter?

Reply:
And we would want a unit test for this in either case...

Reply:
No reason at all - I saw the implementation from `clr-wizard-button`.

About the unit test - it's coming :)